models/engine/db_storage.py

```diff
@@ -97,3 +97,31 @@ def close(self):
         close method on the class Session
         '''
         self.__session.close()
+
+    def get(self, cls, id):
+        '''
+        Method to retrieve one object
+        '''
+        
+
+
+
+    def count(self, cls=None):
+        '''
+        Method to count the number of objects in storage
+        '''
+
+
+    def get(self, cls, id):
+        '''
+        Method to retrieve one object
+        '''
+        key = "{}.{}".format(cls, id)
+        return self.all(cls).get(key) if self.all(cls).get(key) else None
```

Comment:
nice code! I did it this way too, but it's more readable if you put the dictionary in its own variable

Reply:
Thank you for the review and good point about the dictionary variable.